web/source/kmwbase.ts

```diff
@@ -710,7 +710,11 @@ namespace com.keyman {
 
       PKbd = PKbd || this.core.activeKeyboard;
 
-      return com.keyman.osk.VisualKeyboard.buildDocumentationKeyboard(PKbd, argFormFactor, argLayerId, this.osk.computedHeight);
+      // help.keyman.com will set this function in place, so we'll give it priority.
+      let targetHeight = (this['getOskHeight'] as () => number)();
```

Comment:
Is `if (typeof (this['getOskHeight']) == 'function') {` needed?

Reply:
Good catch.  It does appear that we've been similarly type-guarding it everywhere KMW uses it.